pkg/settings/policy.go

```diff
@@ -43,11 +43,11 @@ func (r *PolicyAgent) Load() (err error) {
 	} else {
 		r.CA = ServiceCAFile
 	}
-	r.Limit.Worker, err = getEnvLimit(PolicyAgentWorkerLimit, 25)
+	r.Limit.Worker, err = getEnvLimit(PolicyAgentWorkerLimit, 10)
```

Comment:
In the current design, this throttles concurrent validation request (validation service) for each provider.  As a result, if there are 10 providers, MTV will be sending 250 concurrent validation requests.  Reducing this because I have seen "broken pipe" errors in the log when the provider inventory API starts logging 2-8 seconds response times.
This is a short term fix.  The long term fix is to:
- Perform policy validation using an internal OPA.
- Rate limit the validation request across all providers.
- Rate limit incoming API requests.